hpOneView/resources/hypervisors/hypervisor_cluster_profiles.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+###
+# (C) Copyright [2020] Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+###
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from future import standard_library
+
+standard_library.install_aliases()
+
+
+from hpOneView.resources.resource import Resource
+
+
+class HypervisorClusterProfiles(Resource):
+    """
+    The cluster profile resource manager REST APIs to create, retrieve, modify and delete hypervisor cluster profiles.
+    """
+    URI = '/rest/hypervisor-cluster-profiles'
+
+    DEFAULT_VALUES = {
+        '800': {"type": "HypervisorClusterProfileV3"},
+        '1000': {"type": "HypervisorClusterProfileV3"},
+        '1200': {"type": "HypervisorClusterProfileV3"}
+    }
+
+    def __init__(self, connection, data=None):
+        super(HypervisorClusterProfiles, self).__init__(connection, data)
+
+    def create_virtual_switch_layout(self, data=None, timeout=-1):
+        """Generates vSwitch layout using information specified in the request body.
+
+        Args:
+            data: Fields passed to create the resource.
+            timeout: Timeout in seconds. Wait for task completion by default. The timeout does not abort the operation
+                in OneView; it just stops waiting for its completion.
+
+        Return:
+            Created virtual switch layout.
+        """
+        if not data:
+            data = {}
+
+        vswitch_uri = "{}/virtualswitch-layout".format(self.URI)
+        return self._helper.create(data, vswitch_uri, timeout)
+
+    def get_all(self, start=0, count=-1, filter='', sort='', query='', scope_uris=''):
```

Comment:
just a suggestion, keep the get_all method as the first method of the resource class to align with other resources classes

Reply:
Done